README.md

```diff
@@ -14,17 +14,19 @@ Here is the current status of Python in Azure Functions:
 
 _What are the supported Python versions?_
 
-| Azure Functions Runtime | Python 3.6 | Python 3.7 | Python 3.8 | Python 3.9 |
-|-------------------------|------------|------------|------------|------------|
-| Azure Functions 3.0     | &#x2713;          | &#x2713;          | &#x2713;          | &#x2713;          |
-| Azure Functions 4.0     | &#x2713;          | &#x2713;          | &#x2713;          | &#x2713;          |
```

Comment:
Add  &#x2713; to python 3.10 and 3.11 for v3

Reply:
updated.